patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -124,12 +125,27 @@
  		private static UIWorldSelect _worldSelectMenu = new UIWorldSelect();
  		public static UIManageControls ManageControlsMenu = new UIManageControls();
  		public static UIAchievementsMenu AchievementsMenu = new UIAchievementsMenu();
+@@ -1020,11 +_,11 @@
+ 		public static SoundEffectInstance soundInstanceGrab;
+ 		public static SoundEffect soundPixie;
+ 		public static SoundEffectInstance soundInstancePixie;
+-		public static SoundEffect[] soundItem = new SoundEffect[126];
++		public static SoundWrapper[] soundItem = new SoundWrapper[126];
+ 		public static SoundEffectInstance[] soundInstanceItem = new SoundEffectInstance[126];
+-		public static SoundEffect[] soundNPCHit = new SoundEffect[58];
++		public static SoundWrapper[] soundNPCHit = new SoundWrapper[58];
+ 		public static SoundEffectInstance[] soundInstanceNPCHit = new SoundEffectInstance[58];
+-		public static SoundEffect[] soundNPCKilled = new SoundEffect[63];
++		public static SoundWrapper[] soundNPCKilled = new SoundWrapper[63];
+ 		public static SoundEffectInstance[] soundInstanceNPCKilled = new SoundEffectInstance[63];
+ 		public static SoundEffectInstance soundInstanceMoonlordCry;
+ 		public static SoundEffect soundDoorOpen;
 @@ -1067,7 +_,7 @@
  		public static AudioEngine engine;
  		public static SoundBank soundBank;
  		public static WaveBank waveBank;
 -		public static Cue[] music = new Cue[42];
-+		public static MusicWrapper[] music = new MusicWrapper[42];
++		public static Music[] music = new Music[42];
```

Comment:
This breaks all mods that do anything with the Main.music array. An example would be commented out code in ExampleMod.Load